Jenkinsfile

```diff
@@ -0,0 +1,92 @@
+
+import groovy.json.JsonSlurper
+// This Jenkinsfile is used by Jenkins to run the BioModels step of Reactome's release.
+// It requires that the OrthoinferenceStableIdentifierHistory step has been run successfully before it can be run.
+def currentRelease
+def previousRelease
+pipeline {
+	agent any
+
+	stages {
+  		// This stage checks that an upstream project, AddLinks-Insertion, was run successfully for its last build.
+		stage('Check OrthoinferenceStableIdentifierHistory build succeeded'){
+			steps{
+				script{
+					currentRelease = (pwd() =~ /Releases\/(\d+)\//)[0][1];
+					previousRelease = (pwd() =~ /Releases\/(\d+)\//)[0][1].toInteger() - 1;
```

Comment:
Does it matter that `currentRelease` is not converted to an Integer?

Reply:
No, because ultimately I'm just trying to craft a URL. I just use the integer conversion to subtract so I can check the url from the previous release.